package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@polymer/iron-location",
+  "private": true,
```

Comment:
Why `private: true`? I understand that the idea is to be public in npm.

Reply:
This is separate from the 3.0 move to npm. It'll be public when this code is on npm, this is just adding typings.